### PR TITLE
WiX: Replace default installer images.

### LINKED ISSUE #3502: Replace CD icon with RD icon in MSI installer
The default logo in the MSI installer shows a red square with a retro CD icon inside.

We want to use the Rancher Desktop logo instead.

### PR DESCRIPTION
**Pull request by mook-as**
This customizes the Windows Installer package to use custom images rather than WiX's default red images.

Fixes #3502.  The current images are programmer art.

According to the [documentation](https://learn.microsoft.com/en-us/windows/win32/msi/bitmap-control), we can use Windows bitmap, JPEG, TIFF, PNG, GIF, or whatever HDPhoto is.

The images in this PR in action:
image
image

### CODE REVIEW

File: scripts/lib/installer-win32.tsx [hunk 1]
```diff
@@ -99,6 +99,8 @@ export default async function buildInstaller(workDir: string, appDir: string, de
     '-sice:ICE61',
     `-dappDir=${ appDir }`,
     `-dlicenseFile=${ path.join(appDir, 'build', 'license.rtf') }`,
+    `-dWixUIBannerBmp=${path.join(appDir, 'build', 'wix', 'bannrbmp.png')}`,
```
Comment (Nino-K) [inline scripts/lib/installer-win32.tsx:102]: looks like lint issues?

**Review by Nino-K — COMMENTED** [review]
Looks so much better than the red CD 😄

**Review by Nino-K — APPROVED** [review]
LGTM, but it looks like CI has issues.

**Comment by ericpromislow** [comment]
@Nino-K If all the lint warnings are about Unused eslint-disable directives, you can ignore them. See draft PR #3518 for more info

**Review by ericpromislow — APPROVED** [review]
Pretty good for programmer artwork

**Pull request merged**